components/support/migration/metrics.yaml

```diff
@@ -0,0 +1,550 @@
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483#issuecomment-573971458
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: 2020-10-01
+    lifetime: ping
+
+migration.open_tabs:
+  detected:
+    type: counter
+    description: >
+      Number of detected open tabs.
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483#issuecomment-573971458
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: 2020-10-01
+    lifetime: ping
+  migrated:
+    type: counter
+    description: >
+      Number of migrated open tabs.
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483#issuecomment-573971458
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: 2020-10-01
+    lifetime: ping
+  any_failures:
+    type: boolean
+    description: >
+      Did this migration encounter any failures (exceptions)?
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483#issuecomment-573971458
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: 2020-10-01
+    lifetime: ping
+
+migration.fxa:
+  any_failures:
+    type: boolean
+    description: >
+      Did this migration encounter any failures (exceptions)?
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483#issuecomment-573971458
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: 2020-10-01
+    lifetime: ping
+  failure_reason:
+    type: counter
+    description: >
+      Why this migration failed.
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483#issuecomment-573971458
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: 2020-10-01
+    lifetime: ping
+  failure_reason_rust:
+    type: string
+    description: >
+      Why this migration failed (rust exception).
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483#issuecomment-573971458
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: 2020-10-01
+    lifetime: ping
+  success_reason:
```

Comment:
Nit: I had many questions about why there could be multiple different reasons for success, but they were answered when I found the corresponding values in the code. I wonder if it would be worth fleshing out the descriptions here with a little more detail, to help future analysts avoid similar spelunking.